tests/assets/autoinstall.yaml

```diff
@@ -7,11 +7,9 @@ install:
   grub_options:
     extra_cmdline: "foobarzz"
 
-
-stages:
-  initramfs:
-    - name: "Set user and password"
-      users:
-        kairos:
-          passwd: "kairos"
-      hostname: kairos-{{ trunc 4 .Random }}
+hostname: metal-{{ trunc 4 .MachineID }}
+users:
+  - name: kairos
+    passwd: kairos
+    ssh_authorized_keys:
+      - github:Itxaka
```

Comment:
Probably doesn't hurt to leave here, but just pointing it out in case it wasn't intentional

Reply:
FML :rofl: